test/packages/file_upload_example/file_upload/files/fields/fields.yml

```diff
@@ -0,0 +1,12 @@
+- name: data
+  type: binary
+  description: file chunk
+- name: bid
+  type: keyword
+  description: bid
+  index: false
+- name: last
+  type: boolean
+  description: is the last file chunk
+  index: false
```

Comment:
How will this mapping be different in other packages? Would you have an example?

Reply:
I don't think this would be different between packages.  Each of these docs represents a file chunk which wouldn't have variation.

@pzl let me know if that sounds correct.

cc @kpollich @juliaElastic @joshdover

Reply:
this is the minimum (and mandatory) set of chunk fields. Endpoint likely doesn't need any more. I don't imagine fleet would either. But I haven't thought of any creative use cases outside of that